ubuntu_desktop_installer/lib/welcomepage.dart

```diff
@@ -35,10 +35,10 @@ class _WelcomePageState extends State<WelcomePage> {
     _languageListScrollController = AutoScrollController();
 
     widget.client
-        .fetchKeyboardLayouts(kbdAssetName, Locale(Intl.defaultLocale));
+        .fetchKeyboardLayouts(kbdAssetName, UbuntuDesktopInstallerApp.locale);
     final locale = Intl.defaultLocale;
     for (var i = 0; i < widget.client.languagelist.length; ++i) {
-      if (widget.client.languagelist[i].item1.languageCode == locale) {
+      if (locale.contains(widget.client.languagelist[i].item1.languageCode)) {
```

Comment:
This is where the test is failing (fails locally for me as well). It appears that `locale` is null at this point.

Reply:
I figured it out, `setupAppLocalizations` was not called in test initializations.

So, the `Intl.defaultLocale` gave a null value which lead to the `locale` variable being null too.